tracing-subscriber/Cargo.toml

```diff
@@ -43,6 +44,11 @@ ansi_term = { version = "0.11", optional = true }
 owning_ref = { version = "0.4.0", optional = true }
 chrono = { version = "0.4", optional = true }
 
+# only required by the json feature
+serde_json = { version = "1.0", optional = true }
+serde = { version = "1.0", optional = true }
+tracing-serde = { path = "../tracing-serde", optional = true }
```

Comment:
We'll need to publish a `tracing-serde` release to crates.io before we can publish the changes in this branch — we can't publish a crate that depends on an unpublished crate.

Reply:
True. Do you have a plan when we want this to happen?

Reply:
When these changes land, I can take care of that.